concourse/model/traits/filter.py

```diff
@@ -1,155 +0,0 @@
-# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
This module is still used in delivery-service

Reply:
@8R0WNI3 : it _is_? in that case, I suggest we remove usage asap. Code from concourse package is typically (exception: concourse/client.py) not intended for reuse outside of pipeline-template